Bottom: 4982fbfe20c2e07c3022c8960f9047e1e3448e59
Top:    c6308e970b47e5356d8316dc98a2fd8b03c4eb87
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-21 16:32:12 +0100

Refresh of draw-polygons-in-a-more-intell

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 771c1cb..6ba5c1a 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1270,7 +1270,7 @@ poly_callback_no_clear (const BoxType * b, void *cl)
 static int
 poly_callback_clearing (const BoxType * b, void *cl)
 {
-  struct pin_info *i = (struct pin_info *) cl;
+  struct poly_info *i = (struct poly_info *) cl;
   PolygonType *polygon = (PolygonType *) b;
 
   if (!TEST_FLAG (CLEARPOLYFLAG, polygon))
